### PR TITLE
[netcore] Generate full project setup in gen-xunit-runner

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: netcore/gen-xunit-runner/Program.cs [hunk 1]
```diff
@@ -210,7 +212,6 @@ static int Main (string[] args)
 		var statements = code_main.Statements;
 		statements.Add (new CodeVariableDeclarationStatement (typeof (int), "nrun", new CodePrimitiveExpression (0)));
 		statements.Add (new CodeVariableDeclarationStatement (typeof (int), "nfailed", new CodePrimitiveExpression (0)));
-		statements.Add (new CodeVariableDeclarationStatement (typeof (int), "nskip", new CodePrimitiveExpression (0)));
```
Comment (vargaz) [inline netcore/gen-xunit-runner/Program.cs]: Was this intented ?

File: netcore/gen-xunit-runner/Program.cs [hunk 2]
```diff
@@ -126,7 +128,7 @@ static int Main (string[] args)
 		args = args.Where (s => s != String.Empty).Concat (extra_args).ToArray ();
 
 		// Despite a lot of effort, couldn't get dotnet to load these assemblies from the sdk dir, so copy them to our binary dir
-		File.Copy ($"{sdkdir}/Microsoft.DotNet.PlatformAbstractions.dll", AppContext.BaseDirectory, true);
+//		File.Copy ($"{sdkdir}/Microsoft.DotNet.PlatformAbstractions.dll", AppContext.BaseDirectory, true);
 		File.Copy ($"{sdkdir}/CoreFx.Private.TestUtilities.dll", AppContext.BaseDirectory, true);
```
Comment (vargaz) [inline netcore/gen-xunit-runner/Program.cs]: Was this intended ?

File: netcore/gen-xunit-runner/README.md [hunk 3]
```diff
@@ -4,15 +4,37 @@ This is a small netcore app to convert a corefx test assembly into a console app
...
+dotnet build
 ```
 
+## Tests Execution
+
+This assumes you have sucesfully ran HelloWorld sample before
+
+```
+../../sample/dotnet  --fx-version "3.0.0-preview-27408-5" bin/Debug/netcoreapp3.0/{Name of the test-dll} .dll
+```
```
Comment (vargaz) [inline netcore/gen-xunit-runner/README.md]: Extra space after }

File: netcore/gen-xunit-runner/Program.cs (hunk 1 above)
Comment (marek-safar) [reply netcore/gen-xunit-runner/Program.cs]: yes, it unused

File: netcore/gen-xunit-runner/Program.cs (hunk 2 above)
Comment (marek-safar) [reply netcore/gen-xunit-runner/Program.cs]: yes, it works for me without that too

File: netcore/gen-xunit-runner/README.md (hunk 3 above)
Comment (marek-safar) [reply netcore/gen-xunit-runner/README.md]: will fix it

File: netcore/gen-xunit-runner/README.md [hunk 4]
```diff
@@ -4,15 +4,37 @@ This is a small netcore app to convert a corefx test assembly into a console app
...
+
+The test dll will usually be in the format `System.Runtime.Tests-runner.dll`
+
+
 # Notes
 
 - If xunit can't laod a trait discoverer assembly, it silently ignores the error.
```
Comment (filipnavara) [inline netcore/gen-xunit-runner/README.md]: laod -> load? (on one more place in .cs) ... was bothering me a bit, but didn't seem worth a separate PR.

**Pull request merged**